### PR TITLE
Added vivarium research info to LDL risk effect

### PR DESCRIPTION
**Pull request by ndecleene**

### CODE REVIEW

File: docs/source/gbd2019_models/risk_effects/ldl/index.rst [hunk 1]
```diff
@@ -105,76 +93,80 @@ Vivarium Modeling Strategy
...
 
-.. todo::
-
-	Replace "Risk Outcome Pair #1" with the name of an affected entity for which a modeling strategy will be detailed. For additional risk outcome pairs, copy this section as many times as necessary and update the titles accordingly.
-
-.. todo::
+:ref:`See ischemic heart disease documentation <2019_cause_ihd>`
 
-  Link to existing cause model document or other documentation of the outcome in the risk outcome pair.
+The relative risks apply to the incidence rates of acute myocardial infarction and stable angina. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r367,c493`\) * RR\ :sub:`r367`\). The association was evaluated at the cause level, but the associations should be applied to the incidence rates for both nonfatal components of ischemic heart disease.
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_effects/ldl/index.rst:138]: I was expecting to see the LDL-C exposure in the formula somewhere, e.g.

incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}

File: docs/source/gbd2019_models/risk_effects/ldl/index.rst [hunk 2]
```diff
@@ -105,76 +93,80 @@ Vivarium Modeling Strategy
...
 
-  Complete the following table to list the relative risks for each risk exposure category on the outcome. Note that if there are many exposure categories, another format may be preferable.
+Risk Outcome Pair #2
+++++++++++++++++++++
 
-  Relative risks for a risk factor may be pulled from GBD at the draw-level using :code:`vivarium_inputs.interface.get_measure(risk_factor.{risk_name}, 'relative_risk')`. You can then calculate the mean value as well as 2.5th, and 97.5th percentiles across draws.
+:ref:`See ischemic stroke documentation <2019_cause_ischemic_stroke>`
 
-  The relative risks in the table below should be included for easy reference and should match the relative risks pulled from GBD using the above code. In this case, update the :code:`Note` below to include the appropriate :code:`{risk_name}`.
+The relative risks apply to the incidence rates of acute ischemic stroke. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r367,c495`\) * RR\ :sub:`r367`\).
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_effects/ldl/index.rst:152]: As above, I was expecting the exposure and TMREL to be included in this formula, e.g.

incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}

(But now that I think about it, I think there might be some additional transformation, e.g. dividing max(LDL-C_i - TMREL, 0) by some LDL RR scale like 5.0.

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2019_models/risk_effects/ldl/index.rst (hunk 2 above)
Comment (catherineojohnson) [reply docs/source/gbd2019_models/risk_effects/ldl/index.rst:152]: incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}
^+: indicates max(LDL-C<TMREL, 0)
for SBP, (max(SBP_i - TMREL, 0))/10 bc RR is for a 10 unit increase in Hg

Comment (catherineojohnson) [reply docs/source/gbd2019_models/risk_effects/ldl/index.rst:152]: this is the formula for using a log-linear risk; will need to be updated to address the GBD 2020 approach with splines.

**Review by catherineojohnson — APPROVED** [review]

**Pull request merged**